test/integration/analytics_lakehouse/analytics_lakehouse_test.go

```diff
@@ -48,7 +48,7 @@ func TestAnalyticsLakehouse(t *testing.T) {
 		verifyNoVMs := func() (bool, error) {
 			currentComputeInstances := gcloud.Runf(t, "compute instances list --project %s", projectID).Array()
 			// If compute instances is greater than 0, wait and check again until 0 to complete destroy
```

Comment:
Update to:

`// There should only be 1 compute instance (Dataproc PHS). Wait to destroy if other instances exist.`